config/rbac/metallb_editor_role.yaml

```diff
@@ -17,7 +17,7 @@ rules:
   - update
   - watch
 - apiGroups:
-  - metallb.quay.io/fpaoline/metallboperator
+  - metallb.quay.io/metallb/metallb-operator
```

Comment:
those changes are incorrect this should have been the apiGroup not image ? same as below

Reply:
thanks! tbh I don't remember why I included that commit in this PR so I removed it
we should track these changes here https://github.com/metallb/metallb-operator/pull/14

Reply:
Yep I think this comes from my confusion in setting the parameters when bootstrapping the project with operatorsdk